build-tools/packages/build-cli/src/library/dangerfile.ts

```diff
@@ -47,6 +62,37 @@ export async function dangerfile(): Promise<void> {
 	// there were actual changes to the bundle sizes.  In other cases, we don't post a
 	// message and danger will delete its previous message
 	if (result.comparison === undefined || !bundlesContainNoChanges(result.comparison)) {
+		// Check for bundle size regression
+		let sizeCheck = false;
+		if (result.comparison !== undefined) {
+			for (const bundle of result.comparison) {
+				const totalMetric = bundle.commonBundleMetrics[totalSizeMetricName];
+				const totalParsedSizeDiff =
+					totalMetric.compare.parsedSize - totalMetric.baseline.parsedSize;
+				if (totalParsedSizeDiff > 5120) {
```

Comment:
Probably pull this const out above so we can give it a friendly name, e.g. `sizeWarningThresholdBytes"